### ISSUE TITLE
Homepage redesign

### ISSUE THREAD

**Issue by Klona13** [rr-/TRCustoms#316]
Sidebar:
Like the original, but the level types is now represented by transparent image stripes. When you hover on them, they become opaque. If you click on the right part of the image, it goes to the level area and searches for it. The blue TR1, TR2, etc links are clickable also.

Featured levels:
Now they are big rectangular boxes with a very transparent image from the level page. Information about the level and the cover.
When you hover over it, the transparency of the image decreases and it becomes more visible. Some of the "About" section of the level is there too, enough to fill 5 lines only.

News sidebar:
Has an image stripe like the main sidebar.
[News sidebar explained more in another ticket]

![image](https://user-images.githubusercontent.com/93257509/178830000-a66c4d9f-5ada-4fc9-8f8f-0309e0adb4cb.png)

**Comment by rr-** [comment]
@Klona13 can you please provide the graphics for the new elements

**Comment by Klona13** [comment]
LMK if anything else is required.

TR1
![TR1strip resize](https://user-images.githubusercontent.com/93257509/180607936-55c9898f-f730-4a1b-a95c-c4b74dfb4ec7.png)

TR2
![TR2strip resize](https://user-images.githubusercontent.com/93257509/180607940-82e5f84c-8ce3-4a0a-9bab-d606380cd760.png)

TR3
![TR3strip resize](https://user-images.githubusercontent.com/93257509/180607946-d10645a4-4a63-4619-b591-81d1445c8085.png)

TR4
![TR4strip resize](https://user-images.githubusercontent.com/93257509/180607949-ceaa9510-8f40-4c3c-b64e-25c81d17ae91.png)

TR5
![TR5strip resize](https://user-images.githubusercontent.com/93257509/180607952-df2fdc83-afc8-4fa1-9e26-96f9678bf4bd.png)

News History
![NewsStrip resize](https://user-images.githubusercontent.com/93257509/180608855-fb4d553f-1892-49c5-b028-5aa4ffa5cc88.png)

**Comment by Klona13** [comment]
TEN (when and if it comes out)
![TENstrip resize 2](https://user-images.githubusercontent.com/93257509/181304205-b45e4932-a090-47fa-9a20-0821d62a0858.png)

**Comment by Klona13** [comment]
This shouldn't be allowed. D:
![image](https://user-images.githubusercontent.com/93257509/190871954-2e4f6582-fd76-41c8-8c2e-6de03d500f75.png)

**Comment by rr-** [comment]
I have zero control over this.

**Comment by Klona13** [comment]
^ Correct.

Also there's this issue with the lettuce theme.
![image](https://user-images.githubusercontent.com/93257509/190873579-9cf2db34-e436-4577-afca-98a8c113d186.png)

The color should be the same as here:
![image](https://user-images.githubusercontent.com/93257509/190873592-47c26585-100d-407f-a5be-e0c576b7bae6.png)

**Comment by rr-** [comment]
Fixed.
@Klona13 Unrelated to this ticket but should Metropolis hide the sidebars like Robotic does?

**Comment by Klona13** [comment]
> Fixed. @Klona13 Unrelated to this ticket but should Metropolis hide the sidebar backgrounds like Robotic does?

Metropolis doesn't hide sidebar gbackrounds.

**Comment by rr-** [comment]
>Metropolis doesn't hide sidebar gbackrounds.

Yes, I'm asking if this is intentional?

**Comment by Klona13** [comment]
> > Metropolis doesn't hide sidebar gbackrounds.
> 
> Yes, I'm asking if this is intentional?

yes it is. I thought u were aware as we discussed this some months ago.

**Comment by Klona13** [comment]
Slight clarification about the image strips.
I notice now they get brighter when you hover. It should be like 80% opacity and then it goes to 100% when hovered. The transparency is meant to make them take on the color behind them, blending it a bit so they don't stick out too much.
